common/source/docs/common-vl53l0x-lidar.rst

```diff
@@ -15,6 +15,12 @@ The `VL53L0X <https://www.st.com/en/imaging-and-photonics-solutions/vl53l0x.html
    Support for this lidar was first made available in available in Copter-3.6, Rover-3.3, and Plane-3.9.
    Support the 4m `ST VL53L1X <https://www.st.com/en/imaging-and-photonics-solutions/vl53l1x.html>`__ will be released with Copter-4.0.
 
+
+.. note::
+   
+   This sensor type is suitable for indoor use, however its range and precision is noticeably reduced in bright sunlight conditions, likely beyond usability
```

Comment:
Looks OK.  I might change, "however its range and precision is noticeably reduced in bright sunlight conditions, likely beyond usability" to, "however its range and precision is significantly reduced in bright sunlight conditions and is not recommended for outdoor use".